tests/supervisor/test_execution.py

```diff
@@ -0,0 +1,57 @@
+import asyncio
+import logging
+
+import pytest
+from aleph_message.models import ItemHash
+
+from aleph.vm.conf import settings
+from aleph.vm.controllers.firecracker import AlephFirecrackerProgram
+from aleph.vm.models import VmExecution
+from aleph.vm.orchestrator import metrics
+from aleph.vm.storage import get_message
+
+
+@pytest.mark.asyncio
+async def test_create_execution():
+    """
+    Create a new VM execution and check that it starts properly.
+    """
+
+    settings.FAKE_DATA_PROGRAM = settings.BENCHMARK_FAKE_DATA_PROGRAM
+    settings.ALLOW_VM_NETWORKING = False
+    settings.USE_JAILER = False
+
+    logging.basicConfig(level=logging.DEBUG)
+    settings.PRINT_SYSTEM_LOGS = True
+
+    # Ensure that the settings are correct and required files present.
+    settings.setup()
+    settings.check()
+
+    # The database is required for the metrics and is currently not optional.
+    engine = metrics.setup_engine()
+    await metrics.create_tables(engine)
+
+    vm_hash = ItemHash("cafecafecafecafecafecafecafecafecafecafecafecafecafecafecafecafe")
```

Comment:
should take this value from the settings maybe?

Reply:
This is a dummy hash that can be anything, I thought it would be quite explicit.

I would not put test ids in the global settings. What would you suggest ?

Reply:
My bad I had assumed it had to match the FAKE_INSTANCE_ID in the configuration